aptos-move/block-executor/src/executor.rs

```diff
@@ -141,7 +141,9 @@ where
                     group_key,
                     idx_to_execute,
                     incarnation,
-                    group_ops.into_iter(),
+                    group_ops
+                        .into_iter()
+                        .map(|(tag, (write_op, maybe_layout))| (tag, (write_op, maybe_layout))),
```

Comment:
@vusirikala you haven't addressed the changes from Igor's PR yet, right?

Reply:
Pushed those changes.